README.md

```diff
@@ -3,3 +3,85 @@
 
 Spring Boot Application Reconcilers for Kubernetes
 
+## Spring Boot opinions
```

Comment:
we should add a disclaimer that opinions based on classpath entries may falsely detect an service intent

Reply:
that caveat applies to the whole project :)